### ISSUE TITLE
Subscribe: autocomplete attribute should be applied to the email address field

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#117]
URL: https://hcdev.openplus.ca/en/form/subscribe
User Agent: Chrome
Language: Both

Issue Description: For the email address input field, autocomplete attribute should be applied. Please refer to [Using HTML 5.2 autocomplete attributes](https://www.w3.org/WAI/WCAG21/Techniques/html/H98) and [Input Purposes for User Interface Components](https://www.w3.org/TR/WCAG21/#input-purposes)
Steps to reproduce: Inspect code.
Expected results: autocomplete="email" should be applied to the email address input field.

**Comment by echouin** [comment]
Sent to OP

**Comment by echouin** [comment]
Completed

**Comment by XiuJuanXia** [comment]
autocomplete="email" is applied to the email address input field. Closing this issue.
